cli/prove.ts

```diff
@@ -53,9 +62,31 @@ export const prove = async (
             dataQuery,
         }
 
+        let axiomV2QueryAddress;
+        if (blockhashOracle) {
+            if (broadcaster) {
+                throw new Error("Cannot use both broadcaster and blockhash oracle");
+            }
+            if (!targetChainId) {
+                throw new Error("`targetChainId` is required for blockhash oracle bridge type");
+            }
+            axiomV2QueryAddress = getAxiomV2QueryBlockhashOracleAddress(sourceChainId, targetChainId);
+        } else if (broadcaster) {
+            if (!targetChainId) {
+                throw new Error("`targetChainId` is required for broadcaster bridge type");
+            }
+            if (!bridgeId) {
+                throw new Error("`bridgeId` is required for broadcaster bridge type");
+            }
+            axiomV2QueryAddress = getAxiomV2QueryBroadcasterAddress(sourceChainId, targetChainId, bridgeId);
+        } else {
+            axiomV2QueryAddress = getAxiomV2QueryAddress(sourceChainId);
+        }
+
         let build = await buildSendQuery({
             chainId: sourceChainId,
-            providerUri: provider,
+            rpcUrl: rpcUrlOrCache,
+            axiomV2QueryAddress,   
             dataQuery: res.dataQuery,
             computeQuery: res.computeQuery,
             callback: {
```

Comment:
`buildSendQuery` adds the following optional nested struct into its input object (https://github.com/axiom-crypto/axiom-sdk-client/pull/104/files#diff-3c0ff29938ca23790a6a820f49c35d6f768a90d83393331b48e5adef99f93958R25):

```
crosschain?: {
    bridgeType: BridgeType;
    chainId: string;
    rpcUrl: string;
    bridgeId?: number;
}
```

and we'll need logic to pass in the crosschain object if any item exists (and to throw if one exists but the other doesn't)